arrays-and-slices.md

```diff
@@ -224,12 +223,10 @@ In our case, you can see that having two tests for this function is redundant.
 If it works for a slice of one size it's very likely it'll work for a slice of
 any size \(within reason\).
 
-Go's built-in testing toolkit features a [coverage
-tool](https://blog.golang.org/cover), which can help identify areas of your code
-you have not covered. I do want to stress that having 100% coverage should not
-be your goal, it's just a tool to give you an idea of your coverage. If you have
```

Comment:
`it's just a tool to give you an idea of your coverage`
is essentially a repeat of
`a coverage tool, which can help identify areas of your code you have not covered`
so have reworded to collapse this block without losing any meaning.